### ISSUE TITLE
Lint/ToJSON suggest wrong code

### ISSUE THREAD

**Issue by mrasu** [rubocop-hq/rubocop#6858]
## Expected behavior

With following suggestion at `Lint/ToJSON`, not raise error.
https://github.com/rubocop-hq/rubocop/blob/master/lib/rubocop/cop/lint/to_json.rb#L17

```
@example
  # bad
  def to_json
  end

  # good
  def to_json(_opts)
  end
```

## Actual behavior

Because I get `Lint/ToJSON`, I changed my code like below

```rb
# Before
class Ex
  def to_json
    {a: 1}.to_json
  end
end

# After
class Ex
  def to_json(_opts)
    {a: 1}.to_json
  end
end
```

Then, `Ex.new.to_json` start raising error saying `ArgumentError: wrong number of arguments (given 0, expected 1)`

As talked https://github.com/rubocop-hq/rubocop/pull/6379#discussion_r236021167, document should suggest the argument to be optional

## RuboCop version

```
$ bundle exec rubocop -V
0.66.0 (using Parser 2.6.0.0, running on ruby 2.4.1 x86_64-linux)
```

**Comment by koic** [comment]
I think it was a mistake of multiple arguments. I opened a PR #6859.
